packages/augur-state/package.json

```diff
@@ -34,20 +34,24 @@
     "@augurproject/api": "1.0.0-0",
     "@augurproject/artifacts": "10.0.0-0",
     "@augurproject/core": "2.0.0-0",
+    "@augurproject/ethersjs-provider": "1.0.0-0",
     "@types/async": "2.4.1",
     "@types/web3": "1.0.18",
     "async": "2.6.2",
     "ethereumjs-blockstream": "7.0.0",
     "ethers": "4.0.24",
-    "@augurproject/ethersjs-provider": "1.0.0-0",
     "express": "4.16.4",
     "flexsearch": "0.6.2",
     "ganache-core": "2.4.0",
     "io-ts": "1.4.2",
     "lodash": "4.17.11",
     "postinstall-build": "5.0.3",
-    "pouchdb": "7.0.0",
-    "pouchdb-find": "7.0.0"
+    "pouchdb": "^7.0.0",
+    "pouchdb-find": "7.0.0",
+    "ts-loader": "^5.3.3",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.3.0",
+    "yarn": "^1.15.2"
```

Comment:
Recommend removing the "^" from these. We've had package issues in the past that were caused by not using very specific version numbers, so we've been trying to avoid using the "^" notation in the package.json files.